packages/backend/src/server/api/RateLimiterService.ts

```diff
@@ -30,6 +30,9 @@ export class RateLimiterService {
 		}
 	}
 
+	// Memo: RateLimiter の remaining は 時間内に呼び出せる残り回数
+	// resetはunixtime(エポック秒)
```

Comment:
https://www.npmjs.com/package/ratelimiter

```
Result Object
total - max value
remaining - number of calls left in current duration without decreasing current get
reset - time since epoch in seconds at which the rate limiting period will end (or already ended)
resetMs - time since epoch in milliseconds at which the rate limiting period will end (or already ended)
```